vizro-core/src/vizro/models/__init__.py

```diff
@@ -29,7 +29,7 @@
 Dashboard.update_forward_refs(Page=Page, Navigation=Navigation)
 NavBar.update_forward_refs(NavLink=NavLink)
 NavLink.update_forward_refs(Accordion=Accordion)
-
+Parameter.update_forward_refs(DatePicker=DatePicker)
```

Comment:
Is this still needed? Fine if it is, but it just looks a bit weird since we don't have any other selectors here and don't have `Filter.update_forward_refs`.

Reply:
Removed as no loner needed.